sdks/python/apache_beam/io/avroio.py

```diff
@@ -171,6 +171,7 @@ def __init__(
       min_bundle_size=0,
       desired_bundle_size=DEFAULT_DESIRED_BUNDLE_SIZE,
       use_fastavro=True,
+      with_context=False,
```

Comment:
Should this be called "with_filename" or are you hoping to extend to add more information to the context ? (if latter we should return a more general object instead of a string).

Reply:
Agree! Done!